schema/model.sql

```diff
@@ -25,6 +25,14 @@ create table group_users (
 create index group_users_user_id on group_users(user_id);
 create index group_users_group_id on group_users(group_id);
 
+drop table if exists user_login_histories;
+create table user_login_histories (
+    id int(11) not null auto_increment,
+    user_id int(11) not null,
+    created_at datetime not null,
+    primary key (id, created_at)
```

Comment:
The below limitation forces using `(id, created_at)` as primary key instead of `id`

> All columns used in the table's partitioning expression must be part of every unique key that the table may have, including any primary key. 
> https://dev.mysql.com/doc/refman/5.7/en/partitioning-overview.html

Reply:
I see.